.github/workflows/verify_accessibility_core.yml

```
@@ -0,0 +1,29 @@
+name: Verify Accessibility Core features
+on:
+  pull_request:
+    branches: [master]
+    paths:
+      - 'plugins/catalog/**'
+      - 'plugins/techdocs/**'
+      - 'plugins/scaffolder/**'
+      - 'plugins/search/**'
+
+jobs:
+  lhci:
+    name: Lighthouse
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - name: install dependencies
+        run: |
+          yarn install
```

```suggestion
      - name: yarn install
        uses: backstage/actions/yarn-install@v0.6.3
        with:
          cache-prefix: ${{ runner.os }}-16.x
```

Could use the common action for this right? keeping things a bit speedy